sched/config/defaults.go

```diff
@@ -34,6 +37,15 @@ func DefaultParser() *Parser {
 			"rpc":   &RPCWorkersConfig{},
 			"":      &LocalWorkersConfig{Type: "local"},
 		},
+		Report: map[string]ReportConfig{
+			"default": &DefaultReportConfig{},
+			"": &DefaultReportConfig{
+				Type:       "default",
+				StatsPath:  "/admin/metrics.json",
```

Comment:
Do we know of cases to change StatsPath and/or HealthPath? If not, I'd just not make them configurable.